<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Programme_cinco</class>
 <widget class="QMainWindow" name="Programme_cinco">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1032</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="0" colspan="2">
     <widget class="QTabWidget" name="m_courses">
      <property name="tabPosition">
       <enum>QTabWidget::South</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="m_course1">
       <attribute name="title">
        <string>Course 1</string>
       </attribute>
       <layout class="QGridLayout" name="m_course1Layout">
        <item row="0" column="0">
         <widget class="QScrollArea" name="m_scrollArea1">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>990</width>
             <height>473</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_4"/>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="m_course2">
       <attribute name="title">
        <string>Course 2</string>
       </attribute>
       <layout class="QGridLayout" name="m_course2Layout">
        <item row="0" column="0">
         <widget class="QScrollArea" name="m_scrollArea2">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>990</width>
             <height>473</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_5"/>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="m_course3">
       <attribute name="title">
        <string>Course 3</string>
       </attribute>
       <layout class="QGridLayout" name="m_course3Layout">
        <item row="0" column="0">
         <widget class="QScrollArea" name="m_scrollArea3">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_3">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>990</width>
             <height>473</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_6"/>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="m_course4">
       <attribute name="title">
        <string>Course 4</string>
       </attribute>
       <layout class="QGridLayout" name="m_course4Layout">
        <item row="0" column="0">
         <widget class="QScrollArea" name="m_scrollArea4">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_4">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>990</width>
             <height>473</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_7"/>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="m_course5">
       <attribute name="title">
        <string>Course 5</string>
       </attribute>
       <layout class="QGridLayout" name="m_course5Layout">
        <item row="0" column="0">
         <widget class="QScrollArea" name="m_scrollArea5">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_5">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>990</width>
             <height>473</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_13"/>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="m_course6">
       <attribute name="title">
        <string>Course 6</string>
       </attribute>
       <layout class="QGridLayout" name="m_course6Layout">
        <item row="0" column="0">
         <widget class="QScrollArea" name="m_scrollArea6">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_6">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>990</width>
             <height>473</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout"/>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="m_course7">
       <attribute name="title">
        <string>Course 7</string>
       </attribute>
       <layout class="QGridLayout" name="m_course7Layout">
        <item row="0" column="0">
         <widget class="QScrollArea" name="m_scrollArea7">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_7">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>990</width>
             <height>473</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_14"/>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="m_course8">
       <attribute name="title">
        <string>Course 8</string>
       </attribute>
       <layout class="QGridLayout" name="m_course8Layout">
        <item row="0" column="0">
         <widget class="QScrollArea" name="m_scrollArea8">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_8">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>990</width>
             <height>473</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_15"/>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="m_course9">
       <attribute name="title">
        <string>Course 9</string>
       </attribute>
       <layout class="QGridLayout" name="m_course9Layout">
        <item row="0" column="0">
         <widget class="QScrollArea" name="m_scrollArea9">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_9">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>990</width>
             <height>473</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_16"/>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="m_course10">
       <attribute name="title">
        <string>Course 10</string>
       </attribute>
       <layout class="QGridLayout" name="m_course10Layout">
        <item row="0" column="0">
         <widget class="QScrollArea" name="m_scrollArea10">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_10">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>990</width>
             <height>473</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_17"/>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="m_BDD">
       <attribute name="title">
        <string>BDD</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3"/>
      </widget>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="m_bigClear">
      <property name="text">
       <string>Big Clear</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="m_fillBDD">
      <property name="text">
       <string>Remplissage BDD</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="m_bigStart">
      <property name="text">
       <string>Big Start</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1032</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
